API.md

```diff
@@ -7,6 +7,7 @@
 - `name` - determines the name of the cookie used to store session information. Defaults to _session_.
 - `maxCookieSize` - maximum cookie size before using server-side storage. Defaults to 1K. Set to zero to always use server-side storage.
 - `storeBlank` - determines whether to store empty session before they've been modified. Defaults to _true_.
+- `errorOnCacheNotReady` - will cause yar to throw exception if trying to persist to cache when cache is unavailable. Setting to false will allow application using yar to run uninterrupted if cache is not ready (however sessions will not be saving). Defaults to _false_.
```

Comment:
Error is returned from cache set and get, line 72 and 260 and returned to reply where hapi returns 500.  This option skips those steps to avoid certain exception if the cache is not ready.  When that happens however, the session data does not persist, which is why it shouldn't be default functionality.